data/json/items/vehicle_parts.json

```diff
@@ -399,48 +405,60 @@
         "price_postapoc": 24000
     },
     {
-        "type":"GENERIC",
+        "type":"MAGAZINE",
         "id" : "battery_motorbike",
         "name" : "motorbike battery",
         "name_plural" : "motorbike batteries",
         "description" : "A 12v lead-acid battery used to power smaller vehicles' electrical systems.",
         "weight" : 6000,
         "to_hit" : -2,
         "color" : "light_cyan",
+        "ammo_type": "electricity",
         "symbol" : ":",
         "material" : ["plastic"],
+        "capacity": 2000,
+        "reliability": 10,
+        "flags": [ "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD" ],
         "volume" : 3,
         "bashing" : 2,
         "category" : "veh_parts",
         "price" : 2000
     },
     {
-        "type":"GENERIC",
+        "type":"MAGAZINE",
         "id" : "battery_car",
         "name" : "car battery",
         "name_plural" : "car batteries",
         "description" : "A 12v lead-acid battery used to power car electrical systems.",
         "weight" : 18000,
         "to_hit" : -2,
         "color" : "light_cyan",
+        "ammo_type": "electricity",
         "symbol" : ":",
         "material" : ["plastic"],
+        "capacity": 4500,
+        "reliability": 10,
```

Comment:
This should be the default in `itype.h` as the current default of `0` is useless in all cases